uniffi_bindgen/src/lib.rs

```diff
@@ -443,6 +444,26 @@ fn parse_udl(udl_file: &Utf8Path) -> Result<ComponentInterface> {
     ComponentInterface::from_webidl(&udl).context("Failed to parse UDL")
 }
 
+fn format_code_with_rustfmt(path: &Utf8Path) -> Result<()> {
+    match Command::new("rustfmt").arg(path).status() {
+        Err(e) => {
+            let context = if e.kind() == ErrorKind::NotFound {
+                "formatting was requested, but rustfmt was not found"
+            } else {
+                "unknown error when calling rustfmt"
+            };
+            Err(e).context(context)
+        }
+        Ok(status) => {
```

Comment:
This match arm can instead be `.map_err(|e| { let ctx = match e.kind() { ... }; anyhow!(e).context(ctx) })?`.